test/sslapitest.c

```diff
@@ -54,72 +54,63 @@ struct sslapitest_log_counts {
     unsigned int server_application_secret_count;
 };
 
-static void client_keylog_callback(const SSL *ssl, const char *line) {
+static void client_keylog_callback(const SSL *ssl, const char *line)
+{
     int line_length = strlen(line);
 
     /* If the log doesn't fit, error out. */
-    if ((client_log_buffer_index + line_length) > LOG_BUFFER_SIZE) {
-        printf("No room in client log\n");
+    if (client_log_buffer_index + line_length > sizeof(client_log_buffer)) {
```

Comment:
Should be `> sizeof(client_log_buffer) - 1`, to leave space for the ending NUL byte.